src/main/webapp/definitions/alfresco-scim-v2.yaml

```diff
@@ -0,0 +1,1635 @@
+    - Maximum page size limit enforced
+    - Limited filtering support for Users Endpoint
+  version: "1"
+
+basePath: /alfresco/api/-default-/public/scim/v2
+
+consumes:
+  - application/scim+json
+  - application/json
+
+produces:
+  - application/scim+json
+  - application/json
+
+securityDefinitions:
+  basicAuth:
+    type: basic
+    description: HTTP Basic Authentication
+
+security:
+  - basicAuth: []
+
+parameters:
+  UserId:
+    name: id
+    in: path
+    required: true
+    type: string
+    description: The unique identifier of the user
+    x-example: "2819c223-7f76-453a-919d-413861904646"
+
+  GroupId:
+    name: id
+    in: path
+    required: true
+    type: string
+    description: The unique identifier of the group
+    x-example: "GROUP_EMAIL_CONTRIBUTORS"
+
+  ResourceTypeId:
+    name: id
+    in: path
+    required: true
+    type: string
+    enum: [User, Group, ServiceProviderConfig, ResourceType, Schema]
+    description: The resource type identifier
+    x-example: "User"
+
+  SchemaId:
+    name: id
+    in: path
+    required: true
+    type: string
+    description: The schema identifier (URN)
+    x-example: "urn:ietf:params:scim:schemas:core:2.0:User"
+
+  StartIndex:
+    name: startIndex
+    in: query
+    type: integer
+    minimum: 1
+    default: 1
+    description: The 1-based index of the first query result
+    x-example: 1
+
+  Count:
+    name: count
+    in: query
+    type: integer
+    minimum: 0
+    maximum: 1000
+    default: 1000
+    description: Non-negative integer specifying desired maximum number of query results per page
+    x-example: 10
+
+  FilterUser:
+    name: filter
+    in: query
+    type: string
+    description: |
+      SCIM filter expression.
+      
+      ## Limitations
+      - Users: Only equality filtering on `userName` is supported.
+      
+      ## Examples
+      - `userName eq "john.doe"`
+    x-example: 'userName eq "john.doe"'
+
+  FilterGroup:
+    name: filter
+    in: query
+    type: string
+    description: |
+      SCIM filter expression.
+      
+      ## Examples
+      - `displayName eq "Administrators"`
+    x-example: 'displayName eq "GROUP_EMAIL_CONTRIBUTORS"'
+
+  SortBy:
+    name: sortBy
+    in: query
+    type: string
+    description: |
+      Attribute path for sorting results.
+      
+      **Note:** Sorting is restricted to a single attribute.
+      
+      ## Supported Sort Attributes
+      
+      ### Users
+      - `userName` - Sort by username
+      - `name.familyName` - Sort by family name (last name)
+      - `name.givenName` - Sort by given name (first name)
+      - `emails` or `emails.value` - Sort by email address
+      - `externalId` - Sort by external identifier
+      - `phoneNumbers.work` - Sort by work phone number
+      - `phoneNumbers.mobile` - Sort by mobile phone number
+      - `title` - Sort by user title
+
+      **Note:** Resources without data for the specified sortBy attribute are excluded from the results.
```

Comment:
Can we add the same information in the `/Users/.search` endpoint description (Sorting section)?